web/components/contract/boost-button.tsx

```diff
@@ -58,185 +58,39 @@ export function BoostDialog(props: {
   const { contract, isOpen, setOpen } = props
 
   const [index, setIndex] = useState(0)
-  const [showTabs, setShowTabs] = useState(false)
 
   const [amount, setAmount] = useState<number | undefined>(5000)
 
-  const subsidyDisabled =
-    contract.isResolved ||
-    (contract.closeTime ?? Infinity) < Date.now() ||
-    (contract.mechanism !== 'cpmm-1' && contract.mechanism !== 'cpmm-multi-1')
-
   return (
     <Modal open={isOpen} setOpen={setOpen} size="sm">
       <Col className="bg-canvas-0 gap-2.5  rounded p-4 pb-8 sm:gap-4">
         <Title className="!mb-2">🚀 Boost this question</Title>
-
-        {!showTabs && (
-          <SimpleBoostRow
-            contract={contract}
-            subsidyDisabled={subsidyDisabled}
-            amount={amount}
-            setAmount={setAmount}
-          />
-        )}
```

Comment:
Nice, I thought the simple boost was a little confusing anyway, good to simplify that to just feed promotion